themes/default/content/microsoft-for-startups/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Microsoft for Startups Founders Hub Offer
```

Comment:
This is so awkward (not your fault!)

Maybe ... Microsoft for Startups:  Founders Hub Offer

Reply:
Microsoft specifically requested this change